components/RTM/RealTimeMonitor.vue

```diff
@@ -84,8 +84,8 @@ import moment from 'moment-timezone'
 import ApiRequest from '@/api-request'
 import { InstanceTerritoryControlResponseInterface } from '@/interfaces/InstanceTerritoryControlResponseInterface'
 import { InstancePopulationAggregateResponseInterface } from '@/interfaces/aggregates/instance/InstancePopulationAggregateResponseInterface'
-import { TIME_FORMAT } from '@/constants/Time'
-import { Endpoints } from '@/constants/Endpoints'
+import { TIME_FORMAT } from '@/src/constants/Time'
```

Comment:
This looks wrong, the @ is actually src, so this doesn't make sense. Are you sure you tested this change fully?

Reply:
![image](https://user-images.githubusercontent.com/61867169/180646188-d810eda4-2b96-44df-a2c0-e6187f30c73c.png)
If you do it the other way without the src then you will get an error like so.

However if you do it with the src it doesn't go into src/src it will go to the correct place and you get no errors

Reply:
That's very strange. I'll have a look locally and see why it allows it for some but not others, as right now it's inconsistent

Reply:
Are you sure that @ means src Cause when I did it @ seemed to do the same thing as ~, I haven't been able to find it online anywhere though to confirm that

Reply:
@oslohes123 I know the issue now. Because the new constants submodule folder is not within `/src`, the `@` symbol will not work. You need to move the submodule folder into `src` which is where it should be anyway and then change all the references to the submodule to be `@/ps2alerts-constants/<class>`. 

Alternatively you can put the submodule within `/src/constants/shared` if that feels better, either way the result is the same, you'd reference it like `@/constants/shared/Time` for example.

Reply:
Alright I'll do that at some point just got back from football and kinda feel knackered

Reply:
how about this? 
![image](https://user-images.githubusercontent.com/61867169/180752742-ca5a50a3-bc0e-47c3-be2f-5128430b0414.png)
I've moved the submodule to src and changed the name to ps2alerts-constants whilst keeping the original constants in src